includes/class-mailer.php

```diff
@@ -148,8 +148,12 @@ public static function new_follower( $notification ) {
 	 * @param int   $user_id  The id of the local blog-user.
 	 */
 	public static function direct_message( $activity, $user_id ) {
-		// Check if Activity is public or not.
-		if ( is_activity_public( $activity ) ) {
+		if (
+			is_activity_public( $activity ) ||
+			// Only accept messages that have the user in the "to" field.
+			empty( $activity['to'] ) ||
```

Comment:
should we support empty `to`? is there a usecase for that?

Reply:
This is to avoid a warning in case `to` is not defined.

Reply:
🤦‍♂️